docs/usage/configuration-options.md

```diff
@@ -753,7 +753,7 @@ image: my.old.registry/aRepository/andImage:1.18-alpine
   "customManagers": [
     {
       "customType": "regex",
-      "fileMatch": ["values.yaml$"],
+      "filePatterns": ["/values.yaml$/"],
```

Comment:
we should use glob syntax when appropriate
```suggestion
      "filePatterns": ["values.yaml"],
```

Reply:
@viceice it would need to be `**/values.yaml` if so